### DISCUSSION TITLE
Create a shareable config that extends another one

### DISCUSSION THREAD

**Discussion by noook** [eslint/eslint#15204] [Q&A] [answered]
In our company we use  the framework Vue.js, and we also use the eslint-config associated. We overrid the configuration with custom rules and don't want to update every project's configuration file with new rules.

Instead, we created our shareable config.

In this shareable config, let's name it `@acme/eslint-config` we have 2 files, `vue2.js`, `vue3.js`, both extending the `base.js` file

```js
// vue3.js
module.exports = {
  extends: [
    'plugin:vue/vue3-recommended',
    '@vue/airbnb',
    '@acme/eslint-config/base'
  ],
  // ...
}
```

```jsonc
// package.json
{
  "name": "@acme/eslint-config",
  "version": "0.0.6-beta.1",
  "description": "ACME's ESLint config",
  "license": "MIT",
  "scripts": {},
  "devDependencies": {
    "@vue/eslint-config-airbnb": "^5.3.0",
    "eslint": "^7.31.0",
    "eslint-plugin-vue": "^7.20.0"
  }
}

```

Unfortunately, I can't use it in the project, I can't figure out why. Is it something that is achievable ? Do I need to install all the dependencies in the **root** project ?

**Comment by nzakas** [comment]
> Unfortunately, I can't use it in the project, I can't figure out why. 

what does this mean? Are you seeing an error?

In general, shareable configs must be installed in the project root, even if one config extends another.

**Reply by noook** [reply]
I'm seeing errors such as unable to load the subconfig, but I realized like twenty minutes ago that if I move the plugins from `devDependencies` to `dependencies` it works, I was on my way to write that here !

**Comment by noook** [comment, marked as answer]
UPDATE: You actually need to move the configuration you are extending from to `dependencies` instead of `devDependencies`
